pallets/attestation/src/attestations.rs

```diff
@@ -16,7 +16,6 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use kilt_support::Deposit;
```

Comment:
This change was necessary, otherwise I got the clippy error, that Deposit is shadowed by some other types

Reply:
There was a Deposit struct in the mock module which was used for the tests here. This was causing the shadow. I fixed that in: [2945329](https://github.com/KILTprotocol/kilt-node/pull/655/commits/2945329eef5591962e75495558cd6d1d936ba2be)